remake.yml

```diff
@@ -15,11 +15,29 @@ targets:
   all:
     depends: 3_visualize/out/figure_1.png
   
-  raw_downloads:
-    command: download_nwis_site_data()
+  data_files:
+    command: generate_files()
+
+  1_fetch/out/nwis_01427207_data.csv:
+    command: download_nwis_site_data(target_name)
+
+  1_fetch/out/nwis_01432160_data.csv:
+    command: download_nwis_site_data(target_name)
+
+  1_fetch/out/nwis_01435000_data.csv:
+    command: download_nwis_site_data(target_name)
+
+  1_fetch/out/nwis_01436690_data.csv:
+    command: download_nwis_site_data(target_name)
+
+  1_fetch/out/nwis_01466500_data.csv:
+    command: download_nwis_site_data(target_name)
+
+  1_fetch/out/nwis_01477050_data.csv:
+    command: download_nwis_site_data(target_name)
 
   site_data:
-    command: import_nwis_data(raw_downloads)
+    command: import_nwis_data(data_files)
```

Comment:
this is close, but since data_files is the names of the files, it won't be reflected as changed if the `1_fetch/out/nwis_01466500_data.csv` data in there change. 

To check if the pipeline is true to the updates, see if changing the `startDate="2013-05-01"` and make sure that the files re-download. If not, consider something like this:

```yaml
  site_data:
    command: import_nwis_data('1_fetch/out/nwis_01427207_data.csv',
      '1_fetch/out/nwis_01432160_data.csv',
      '1_fetch/out/nwis_01435000_data.csv',
      '1_fetch/out/nwis_01436690_data.csv',
      '1_fetch/out/nwis_01466500_data.csv',
      '1_fetch/out/nwis_01477050_data.csv')
```
with a mod to your function like this:
```r
import_nwis_data <- function(...) {
  download_files <- c(...)
```

Reply:
Oh, that's a really good point 🤦 . Sorry - I wasn't thinking ahead there.

It definitely works WAY better now. Especially because NWIS is queried separately for each site, so if the connection fails for site C, you at least still retain the data that had been successfully queried for sites A and B (and don't have to rebuild those when you re-run scmake() !!)